torch/csrc/jit/python/python_sugared_value.h

```diff
@@ -176,6 +176,10 @@ struct VISIBILITY_HIDDEN ModuleValue : public SugaredValue {
       const SourceRange& loc,
       Function& m);
 
+  std::shared_ptr<SugaredModuleDict> getSugaredNamedBufferDict(
```

Comment:
Probably we should rename `SugaredModuleDict` something more general, to reflact that it is not just being used as for `named_modules()` anymore.

Reply:
Agreed. Any suggestions?

Reply:
SugaredDict?